chips/upd765.h

```diff
@@ -638,10 +658,10 @@ static inline uint8_t _upd765_read_status(upd765_t* upd) {
             status |= UPD765_STATUS_CB|UPD765_STATUS_RQM;
             break;
         case UPD765_PHASE_EXEC:
-            status |= UPD765_STATUS_CB|UPD765_STATUS_EXM|UPD765_STATUS_DIO|UPD765_STATUS_RQM;
+            status |= UPD765_STATUS_CB|UPD765_STATUS_EXM|dio|UPD765_STATUS_RQM;
```

Comment:
The DIO flag handling seems to be more involved (also see MAME: https://github.com/mamedev/mame/blob/b4ad26d9802c6191470d6dc6b4ab081022d22cf0/src/devices/machine/upd765.cpp#L494-L503)